### PR TITLE
Gracefully handle string labels in wandb.sklearn.plot.classifier.calibration_curve

### PR DESCRIPTION
**Pull request by acrellin**
Description
-----------
This patch addresses a bug that would cause `wandb.sklearn.plot_classifier`
to error out during the calibration curve generation if a list of string labels is
provided (the element-wise comparison fails in this case, and returns a scalar
boolean). Instead of erroring out, we now warn that this plot is being skipped.

Testing
-------
How was this PR tested?

Checklist
-------
- Name PR "[WB-NNNN][WB-MMMM] Add support for..." similar to entries in CHANGELOG.md
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

**Review by dmitryduev — APPROVED** [review]
👍🏻 Like good ol' days, @acrellin ;)
